torch/_inductor/codecache.py

```diff
@@ -1832,6 +1832,11 @@ def _pad_to_alignment(raw_bytes: bytes) -> bytes:
                 "linux": _compile_consts_linux,
                 "darwin": _compile_consts_darwin,
             }[sys.platform](aot_constants)
+            other_obj_files = []
+            for entry in CUDACodeCache.cache.values():
+                if entry.output_path.endswith(".o"):
+                    other_obj_files.append(entry.output_path)
+            other_o = " ".join(other_obj_files)
```

Comment:
also pretty hacky imo.. maybe I can create another abstraction to easily grab object files for CUTLASS/CK wrapper functions.

Reply:
Is there a cache instance per model, or will it also pick up object files from different models?

Reply:
it's cached per model

and sg will add comments or think of a better name

Reply:
One caveat here is, the CK backend uses ROCmCodeCache, so there has to be a way to switch to a correct cache object